bin/create

```diff
@@ -23,14 +23,15 @@ var create = require('./lib/create');
 var args  = require('./lib/simpleargs').getArgs(process.argv);
 
 if (args['--help'] || args._.length === 0) {
-    console.log('Usage: ' + path.relative(process.cwd(), path.join(__dirname, 'create')) + ' <path_to_new_project> <package_name> <project_name> [<template_path>] [--link]');
+    console.log('Usage: ' + path.relative(process.cwd(), path.join(__dirname, 'create')) + ' <path_to_new_project> <package_name> <project_name> <activity_name> [<template_path>] [--link]');
```

Comment:
I wasn't sure of the best way to make both template_path and activity_name optional. I explored using --activity-name <activity_name> using nopt, which cordova-ios uses for arguments.